kik_unofficial/client.py

```diff
@@ -865,38 +868,38 @@ def data_received(self, data: bytes):
         # Handle empty packets wont get caught by logic below
         if data == b" ":
             self.loop.call_soon_threadsafe(self.api._on_new_data_received, data)
-
-        # Handle </pong> messages sometimes they can come in a <ack> message which would be thrown away.
-        pong_match = re.search(b'<pong/>', data)
-        if pong_match:
-            pong_data = pong_match.group()
-            self.loop.call_soon_threadsafe(self.api._on_new_data_received, pong_data)
+        elif re.search(b'<pong/>', data):
+            self.loop.call_soon_threadsafe(self.api._on_new_data_received, data)
+        elif not self.is_multi_packet(data):
+            self.loop.call_soon_threadsafe(self.api._on_new_data_received, data)
```

Comment:
Let's add a comment that says that this is the usual flow (no multi packet, no pong, not empty)

Reply:
Added

Reply:
Let's just put that after the `elif` line...